### ISSUE TITLE
vue-docgen-api: does not infer the the slot binding descriptions when the prop names are kebab cased

### ISSUE THREAD

**Issue by sevilyilmaz** [vue-styleguidist/vue-styleguidist#1229]
**Current behavior**

The scoped-slot that that a component has three props one of which is kebab-cased (hypen-cased) because of the linting requirements of the library. `vue-docgen-api` fails to infer the description of the prop that is kebab-cased.

```vue
<!--
  @slot for customizing individual days.
    @binding {date} date The date instance of the day
    @binding {number} day The day number (e.g 21)
    @binding {string} aria-label aria-label for the day
-->
<slot
  name="day"
  :date="date"
  :day="day"
  :aria-label="ariaLabel"
></slot>
```

**To reproduce**

I used [an existing test](https://github.com/vue-styleguidist/vue-styleguidist/blob/dev/packages/vue-docgen-api/src/template-handlers/__tests__/slotHandler.ts#L126-L151) to reproduce. The test below fails with;

```
    expect(received).toMatchObject(expected)

    - Expected  - 1
    + Received  + 0

    @@ -6,9 +6,8 @@
        Object {
          "description": "text of the menu item",
          "name": "otherItem",
        },
        Object {
    -     "description": "aria-label for the day",
          "name": "aria-label",
        },
      ]
```

```js
it('should detect explicit bindings and allow their documentation', () => {
	const ast = parse(
		[
			'<div title="a list of item with a scope" >',
			'	<!--',
			'		@slot Menu Item footer',
			'		@binding {object} item menu item',
			'		@binding {string} otherItem text of the menu item',
			'		@binding {number} aria-label aria-label for the day',
			'	-->',
			'  <slot name="bound" v-for="item in items" :item="item" :otherItem="valueGiven" :aria-label="ariaLabel" />',
			'</div>'
		].join('\n')
	)
	traverse(ast.children[0], doc, [slotHandler], ast.children, { functional: false })
	const slots = doc.toObject().slots || []
	expect(slots.filter(s => s.name === 'bound')[0].bindings).toMatchObject([
		{
			name: 'item',
			description: 'menu item'
		},
		{
			name: 'otherItem',
			description: 'text of the menu item'
		},
		{
			name: 'aria-label',
			description: 'aria-label for the day'
		}
	])
})
```

**Expected behavior**

It should return the binding description regardless of the casing of the props.
